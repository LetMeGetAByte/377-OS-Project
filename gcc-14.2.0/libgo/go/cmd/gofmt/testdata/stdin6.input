version https://git-lfs.github.com/spec/v1
oid sha256:00a33dbbd65c7399c465acb9b09266c7df11791671e28615fa017ecbb31246c4
size 217
