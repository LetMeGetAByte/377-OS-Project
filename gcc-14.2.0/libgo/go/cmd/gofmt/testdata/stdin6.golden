version https://git-lfs.github.com/spec/v1
oid sha256:7a6b7393ed02db6048866bdea4c3539c8f1f8aa562b5e1ca6f57a478b310f7e2
size 199
